File: C/Assignment 5/inverted pyramid.c
// inverted pyramid
#include<stdio.h>
void main()
{
	int no=5;
	for(int row=no;row>=1;row--)
	{
		for(int space=1;space<row;space++)
		{
			printf(" ");
		}
		for(int star=1;star<=no-row+1;star++)
		{
			printf("* ");
		}
	printf(" \n");
	}
}

File: OOP/Assignment 5/Inverted_pyramid.java
class Inverted_pyramid
{
public static void main(String args[])
{
	int no=5;
	for(int row=no;row>=1;row--)
	{
		for(int space=1;space<row;space++)
		{
			System.out.printf(" ");
		}
		for(int star=1;star<=no-row+1;star++)
		{
			System.out.printf("* ");
		}
	System.out.printf(" \n");
	}
}
}